### DISCUSSION TITLE
Future plans for Chakra

### DISCUSSION THREAD

**Discussion by PasqualePuzio** [reflex-dev/reflex#2647] [General]
Hi,

first of all congrats on the latest release, keep up the excellent work!

I've seen that core components are now based on Radix, so I was wondering if and for how long Chakra will still be supported. What are your plans about it?

Since we're building quite a large app with Reflex, if you plan at some point to remove and drop support for Chakra, we may consider to invest some time right away to migrate to Radix.

Thanks

**Comment by masenf** [comment]
No immediate plans to drop support for chakra, but we might move it to a separate installable package outside of the core.
